README.md

```diff
@@ -9,7 +9,7 @@ phys2bids
 [![Join the chat at Gitter: https://gitter.im/physiopy/phys2bids](https://badges.gitter.im/physiopy/phys2bids.svg)](https://gitter.im/physiopy/phys2bids?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=body_badge)
 [![codecov](https://codecov.io/gh/physiopy/phys2bids/branch/master/graph/badge.svg)](https://codecov.io/gh/physiopy/phys2bids)
 
-[![Build Status: Linux](https://travis-ci.org/physiopy/phys2bids.svg?branch=master)](https://travis-ci.org/physiopy/phys2bids)
+[![CircleCI](https://circleci.com/gh/physiopy/phys2bids.svg?style=shield)](https://circleci.com/gh/physiopy/phys2bids)
```

Comment:
```suggestion
[![CircleCI](https://circleci.com/gh/physiopy/phys2bids.svg?branch=master&style=shield)](https://circleci.com/gh/physiopy/phys2bids)
```

To only show the status on the master branch.

Reply:
I thought that was the default. Thanks!